azurelinuxagent/common/cgroups.py

```diff
@@ -30,6 +32,15 @@
 WRAPPER_CGROUP_NAME = "Agent+Extensions"
 METRIC_HIERARCHIES = ['cpu', 'memory']
 MEMORY_DEFAULT = -1
+UNLIMITED_INSTANCES = ['customscript', 'runcommand']
```

Comment:
Consider putting this list of "extensions which are exceptions" into the waagent.conf file. This allows customers to make exceptions for extensions we don't know about today without requiring an agent re-release.

Reply:
Good call. I will open a new PR for that change so I can get some coverage on this one overnight.